packages/flutter/test/cupertino/refresh_test.dart

```diff
@@ -1327,48 +1327,62 @@ void main() {
         );
     }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }));
 
-    testWidgets('buildSimpleRefreshIndicator dark mode', (WidgetTester tester) async {
-      const CupertinoDynamicColor color = CupertinoColors.inactiveGray;
+    testWidgets('buildRefreshIndicator progress', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        Directionality(
+          textDirection: TextDirection.ltr,
+          child: Builder(
+            builder: (BuildContext context) {
+              return CupertinoSliverRefreshControl.buildRefreshIndicator(
+                context,
+                RefreshIndicatorMode.drag,
+                10, 100, 10,
+              );
+            },
+          ),
+        ),
+      );
+
+      // If the user has dragged less than the threshold, there is no activity indicator
+      // returned. Instead, an empty container is returned until the threshold is passed.
+      expect(find.byType(Container), findsOneWidget);
```

Comment:
One test is failing because the `Container` is now gone. Could you fix it?

Reply:
hmm... that's weird. Somehow I didn't push that change up. Sorry... changes incoming.